xbmc/ContextMenuManager.cpp

```diff
@@ -72,7 +73,12 @@ void CContextMenuManager::Init()
       std::make_shared<CONTEXTMENU::CMarkWatched>(),
       std::make_shared<CONTEXTMENU::CMarkUnWatched>(),
   };
+
   ReloadAddonItems();
+
+  std::vector<std::shared_ptr<IContextMenuItem>> pvrItems = m_pvrMgr.GetContextMenuItems();
```

Comment:
I want to stick with with <code>CPVRContextMenuManager </code> for decoupling purposes. I'm trying to keep all pvr code in xbmc/pvr. But you are perfectly right. It makes perfect sense to get rid of the dependency to the "heavy" <code>CPVRMananager</code>. I will change the PR accordingly. Thanks for the hint.